builtin/logical/pki/path_tidy.go

```diff
@@ -561,6 +579,108 @@ func pathTidyStatus(b *backend) *framework.Path {
 }
 
 func pathConfigAutoTidy(b *backend) *framework.Path {
+	autoTidyResponseFields := map[string]*framework.FieldSchema{
```

Comment:
👏 Thank you for removing duplicate code!

Reply:
Ditto on this - thank you very much